site/docs/v1/tech/messengers/generic-messenger.adoc

```diff
@@ -80,3 +80,43 @@ In these fields, you can add custom key and value pairs to augment your headers.
 image::messengers/configuration-test-generic.png[Test Configuration Generic, role=top-cropped, width=1200]
 
 You also can test your generic messenger configuration. By hitting `Test generic configuration` FusionAuth will fire a JSON message to the messenger just created to ensure everything is set up correctly.
+
+== Writing the Generic Message Receiver
+
+You need to write a server side component, a receiver, which will be deployed at a URL and send the actual messages. This component can be written in any language that can consume a JSON message.
+
+=== Request 
+
+The request to your endpoint will be delivered as JSON.
+
+[source,json]
+.Example Request JSON
+----
+include::docs/src/json/messengers/generic-message-request.json[]
+----
+
+The components of this message include:
+
+* [field]#phoneNumber#, the number of the user to which to send the message.
+* [field]#textMessage#, the message to send. It is built from the configured link:/https://fusionauth.io/docs/v1/tech/email-templates/message-templates/[message template] and will be localized.
+* [field]#type#, which is always `SMS`.
+
+When your application receives this message, it should send the notification to the phone number provided. 
```

Comment:
Or proxy to the 3rd party that is actually going to send the message? I would assume it is unlikely the builder of this proxy/component will be the one sending the message. 

The proxy could call another SDK to send the message, or build another API request to a 3rd party.